discretisedfield/region.py

```diff
@@ -184,43 +359,51 @@ def edges(self):
         (5, 15, 20)
 
         """
-        return dfu.array2tuple(np.abs(np.subtract(self.p1, self.p2)))
+        return np.abs(np.subtract(self.pmin, self.pmax))
 
-    @functools.cached_property
-    def centre(self):
-        r"""Centre point.
+    @property
+    def center(self):
+        r"""Center point.
 
-        Centre point is computed as the middle point between region's points
+        Center point is computed as the middle point between region's points
         with minimum and maximum coordinates:
 
         .. math::
 
-            \mathbf{p}_\text{centre} = \frac{1}{2} (\mathbf{p}_\text{min}
+            \mathbf{p}_\text{center} = \frac{1}{2} (\mathbf{p}_\text{min}
             + \mathbf{p}_\text{max}).
 
         Returns
         -------
-        tuple (3,)
+        numpy.ndarray (3,)
 
-            Centre point :math:`(p_c^x, p_c^y, p_c^z)`.
+            Center point :math:`(p_c^x, p_c^y, p_c^z)`.
 
         Examples
         --------
-        1. Getting the centre point.
+        1. Getting the center point.
 
         >>> import discretisedfield as df
         ...
         >>> p1 = (0, 0, 0)
         >>> p2 = (5, 15, 20)
         >>> region = df.Region(p1=p1, p2=p2)
         ...
-        >>> region.centre
+        >>> region.center
         (2.5, 7.5, 10.0)
 
         """
-        return dfu.array2tuple(0.5 * np.add(self.pmin, self.pmax))
+        return 0.5 * np.add(self.pmin, self.pmax)
 
-    @functools.cached_property
+    @property
+    def centre(self):
+        warnings.warn(
+            "Spelling has been changed to American spelling; use 'center' instead.",
+            FutureWarning,
```

Comment:
`FutureWarning` or `DeprecationWarning`?

Reply:
I feel `DeprecationWarning` is stronger than `FutureWarning`. If `centre` will be removed "soon", then I would go for `DeprecationWarning`. But I am not sure if everyone looks at it that way?

Reply:
We had an in-person discussion and decided to raise an exception right away.